kitsune/search/tests/test_api.py

```diff
@@ -122,15 +106,15 @@ def test_invalid_numbers(self):
             'q': 'search',
         })
         eq_(res.status_code, 400)
-        eq_(res.data['detail'], {
-            'max_questions': [u'Enter a whole number.'],
-            'max_documents': [u'Enter a whole number.'],
+        eq_(res.data, {
```

Comment:
Yes, that is true. DRF changed the way it rendered errors normally. The "everything is fine" case is the same, but they unpacked the error responses a bit, moving from

```json
{
    "detail": {
        "field": ["list of errors"],
    }
}
```

to the simple

```json
{
    "field": ["list of errors"],
}
```

Considering where this API is used, I think that is an ok change to make, although technically it represents a breaking change. I'm not sure how we could force DRF to use the previous style of error handling (and I'm a bit grumpy that they changed it).

What do you think? Should we try and bend DRF into submission and make it use the old way?